### PR TITLE
Add code marks where necessary

### PR DESCRIPTION
**Pull request by geekysrm**

### CODE REVIEW

File: Development-Guidelines/Coding-Standards/jquery-guidelines.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ So essentially, we'll be creating 2 plugins, one to instantiate it and one to re
 
 ### Creating the plugins
 
+```
```
Comment (jmayntzhusen) [inline Development-Guidelines/Coding-Standards/jquery-guidelines.md]:
```suggestion
```javascript
```

If we are adding codeblocks we may as well also add syntax highlighting 😉

**Comment by geekysrm** [comment]
Please check @jmayntzhusen

**Comment by geekysrm** [comment]
Also quick question: How will we be contacted for the swag 😍 opportunity as mentioned here (https://umbraco.com/blog/happy-hacktoberfest/)?

**Comment by jmayntzhusen** [comment]
Thanks for the contribution @geekysrm !

Information on the swag will come late October / early November as there are still a few moving pieces 😉

**Pull request merged**